Apparently this doesn't make sense. Otherwise the queue gets disabled as
soon as it's getting empty and can only be resurrected by a driver
restart.

Signed-off-by: Phil Sutter <n0-1@freewrt.org>
---
 drivers/net/korina.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

--- a/drivers/net/korina.c
+++ b/drivers/net/korina.c
@@ -272,7 +272,6 @@ static int korina_send_packet(struct sk_
 			/* Move tail */
 			lp->tx_chain_tail = chain_next;
 			lp->tx_chain_status = desc_filled;
-			netif_stop_queue(dev);
 		} else {
 			/* Update tail */
 			td->control = DMA_COUNT(length) |
